src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -114,29 +125,253 @@ Template:
-| Inplane T1         | inplaneT1        | T1-weighted anatomical image matched to functional acquisition                                                                                    |
-| Inplane T2         | inplaneT2        | T2-weighted anatomical image matched to functional acquisition                                                                                    |
-| Angiography        | angio            |                                                                                                                                                   |
+        sub-<label>[_ses-<label>][_acq-<label>][_part-<label>][_echo-<index>][_fa-<index>][_inv-<index>][_mt-<on/off>][_ce-<label>][_rec-<label>][_run-<index>]_<suffix>.nii[.gz]
+        sub-<label>[_ses-<label>][_acq-<label>][_part-<label>][_echo-<index>][_fa-<index>][_inv-<index>][_mt-<on/off>][_ce-<label>][_rec-<label>][_run-<index>][_mod-<suffix>]_defacemask.nii[.gz]
+```
+
+The term anatomical imaging data pertains to a broad range of MRI applications that provide structural 
+information about (brain) anatomy, but differ by the nature of the data they contain. Anatomical
+(structural) data for a participant may refer to three types of data:
+1. a single image with specific weighting on an arbitrary scale (e.g. a 3D high resolution T1-weighted image).
+This type is most commonly used in neuroimaging applications.
+2. a group of images acquired within a single protocol for the purpose of improving contrast 
+characteristics (e.g., a set of multi-echo anatomical GRE images) or calculating quantitative maps
+(e.g. four 3D volumes provided as an input to an MP2RAGE calculation) 
+3. a quantitative map of which the intensities are put on an absolute scales (e.g., seconds for a T1 map).
+These images are generally produced by some mathematical operation on images from class 2.
+
+#### Conventional structural acquisitions
+
+Anatomical images of this type typically refer to a volumetric high-resolution 
+dataset, representing the measured MRI signal in an arbitrary scale of "gray
+shades". Contrast factor of these grayscale images depend on the relative contribution
+of inherent tissue parameters (e.g. `T1`, `T2` and `PD`) to the measured signal. 
+
+Contribution weights of these parameters are determined by the type of acquisition
+sequence (e.g. `spin-` or `gradient-echo`) and the setting of various parameters
+(e.g. `Repetition Time`, `Echo Time`, `Inversion Time` and `Flip Angle`). For 
+example, in a spoiled gradient-echo scan, keeping the repetition and the echo time 
+short with a relatively large flip angle increases the contribution of `T1` effects
+to the image contrast, yielding a primarily T1-weighted image.
+
+#### Grouped scan collections for contrast improvement or quantitative map calculation
+
+A group of anatomical scans may be collected to enhance certain contrast features,
+such as to calculate a weighted average of multi-echo gradient echo (`MEGRE`) images,
```

Comment:
Wouldn't multi-echo GRE images be acquired in the same scan?

Reply:
Fair point. This is actually also true for, e.g., `MP2RAGE` (multiple inversion times after the same pulse).

Maybe 

```
A group of multiple anatomical images may be collected to enhance certain contrast features,
```

What do you think?